src/libraries/System.Runtime/tests/System.Runtime.Tests/System/ExitCodeTests.Unix.cs

```diff
@@ -58,8 +58,8 @@ public void SigTermExitCode(int? exitCodeOnSigterm)
                 bool exited = process.WaitForExit(RemoteExecutor.FailWaitTimeoutMilliseconds);
                 Assert.True(exited);
 
-                // Check exit code
-                Assert.Equal(exitCodeOnSigterm ?? 128 + SIGTERM, process.ExitCode);
+                // Check that the exit code is 143 (128 + SIGTERM).
+                Assert.Equal(143, process.ExitCode);
```

Comment:
This test is was written to check `AppDomain.CurrentDomain.ProcessExit` is called on `SIGTERM`. The `exitCodeOnSigterm` variable no longer makes sense.